src/dataflow/load.js

```diff
@@ -43,7 +43,7 @@ export function request(target, url, format) {
         df.ingest(target, data, format);
       },
       function(error) {
-        df.error('Loading failed: ' + url, error);
+        df.error('Loading failed: ' + (typeof url === 'string' ? url : JSON.stringify(url)), error);
```

Comment:
agree - I was thinking about that too -- in the first case, are you ok with URL being quoted?  as for the second case, sure, will wrap it in try catch - i was thinking that the only code that calls it is coming from a JSON parser, but you are right, vega could have been given as an object.